starter-core/build.gradle

```diff
@@ -14,23 +14,26 @@ sourceSets {
 }
 
 dependencies {
-    api platform("io.micronaut:micronaut-bom:$micronautVersion")
+    api platform("io.micronaut.platform:micronaut-platform:$micronautVersion")
```

Comment:
Should we be pulling platform here?

Or micronaut-core-bom?

Reply:
platform.